crates/chain/src/indexed_tx_graph.rs

```diff
@@ -61,6 +61,13 @@ impl<A: Anchor, I: Indexer> IndexedTxGraph<A, I> {
 
         self.graph.apply_changeset(graph);
     }
+
+    /// Determines the changeset between self and the empty [`IndexedTxGraph`].
```

Comment:
Do we not need to write self as `self`?

Reply:
Nit: This is more descriptive of how this method should be used.

```suggestion
    /// Determines the [`ChangeSet`] between `self` and an empty [`IndexedTxGraph`].
    ///
    /// The returned `ChangeSet` can be applied to an empty `IndexedTxGraph` to recover the current
    /// state of `self`.
```